app/assets/stylesheets/views/mod-center.scss

```diff
@@ -116,14 +117,25 @@
       border: none;
     }
 
-    .article-iframe {
+    .article-referrer-heading {
+      padding: var(--su-4);
       width: 100%;
-      height: 650px; // height is arbitrary, will be removed for modal-esque view
     }
 
-    .actions-panel-iframe {
-      width: 60%;
-      height: 650px; // height is arbitrary, will be removed for modal-esque view
+    .iframes-container {
+      display: flex;
+      height: 650px;
+      width: 100%;
+
+      .article-iframe {
+        width: 100%;
+        height: 650px; // height is arbitrary, will be removed for modal-esque view
```

Comment:
Also, what does this comment mean? (wasn't really able to wrap my head around it)